packages/mdc-textfield/input/README.md

```diff
@@ -0,0 +1,67 @@
+<!--docs:
+title: "Text Field Input"
+layout: detail
+section: components
+excerpt: "The input element is where the user enters text"
+iconId: input
+path: /catalog/input-controls/text-fields/input/
+-->
+
+# Text Field Input
+
+The input element is where the user enters text.
+
+## Design & API Documentation
+
+<ul class="icon-list">
+  <li class="icon-list-item icon-list-item--spec">
+    <a href="https://material.io/guidelines/components/text-fields.html#text-fields-states">Material Design guidelines: Text Fields States</a>
```

Comment:
I think this link

https://material.io/guidelines/components/text-fields.html#text-fields-field-types

might be a better place to link to

Reply:
Done